src/conninode.cpp

```diff
@@ -102,8 +102,11 @@ void addtoconninode(char *buffer) {
     if ((in6_local.s6_addr32[0] == 0x0) && (in6_local.s6_addr32[1] == 0x0) &&
         (in6_local.s6_addr32[2] == 0xFFFF0000)) {
       /* IPv4-compatible address */
-      result_addr_local = *((struct in6_addr *)&(in6_local.s6_addr32[3]));
-      result_addr_remote = *((struct in6_addr *)&(in6_remote.s6_addr32[3]));
+      memset(&result_addr_local,0,sizeof(in6_addr));
```

Comment:
Hmm, that warning doesn't really make sense - I'm tempted to add `-Wno-missing-field-initializers`